### PR TITLE
Sync various sections

### PR DESCRIPTION
**Pull request by saundefined**

### CODE REVIEW

File: reference/datetime/functions/timezone-name-from-abbr.xml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 <!-- $Revision$ -->
-<!-- EN-Revision: b95d28e6ec86e4a71e012737d36ebdc1cf009180 Maintainer: jpauli Status: ready -->
+<!-- EN-Revision: 5c951013ca04161992efed8b86fb40f55669958e Maintainer: jpauli Status: ready -->
 <!-- Reviewed: yes -->
 
 <refentry xml:id="function.timezone-name-from-abbr" xmlns="http://docbook.org/ns/docbook">
  <refnamediv>
   <refname>timezone_name_from_abbr</refname>
-  <refpurpose>Retourne le nom du fuseau horaire à partir de son abréviation</refpurpose>
+  <refpurpose>Retourne le nom d'un fuseau horaire en devinant l'abréviation et le décalage TUC</refpurpose>
```
Comment (saundefined) [inline reference/datetime/functions/timezone-name-from-abbr.xml:9]:
```suggestion
  <refpurpose>Retourne le nom d'un fuseau horaire en devinant l'abréviation et le décalage UTC</refpurpose>
```

Or should I use "UTC" instead? TUC retrieved from [wiki](https://fr.wikipedia.org/wiki/Temps_universel_coordonn%C3%A9) :)

Comment (adoy) [reply reference/datetime/functions/timezone-name-from-abbr.xml:9]: You should use UTC and change the sentence to 
```
Retourne le nom d'un fuseau horaire à partir de son abréviation et de son décalage UTC.
```

File: reference/datetime/functions/idate.xml [hunk 2]
```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- $Revision$ -->
-<!-- EN-Revision: 5e865c822bc21a97228facd06533a3c93ba5edad Maintainer: yannick Status: ready -->
+<!-- EN-Revision: 5c951013ca04161992efed8b86fb40f55669958e Maintainer: yannick Status: ready -->
 <!-- Reviewed: yes Maintainer: yannick -->
 <refentry xml:id="function.idate" xmlns="http://docbook.org/ns/docbook">
  <refnamediv>
   <refname>idate</refname>
-  <refpurpose>Formate une date/heure locale en tant qu'entier</refpurpose>
+  <refpurpose>Formate d'une partie de la date/heure locale en tant qu'entier</refpurpose>
```
Comment (adoy) [inline reference/datetime/functions/idate.xml:8]:
```suggestion
  <refpurpose>Formate une partie de l'heure/date locale en tant qu'entier</refpurpose>
```

**Review by adoy — COMMENTED** [review]
Thanks !

**Review by adoy — APPROVED** [review]
lgtm

**Pull request merged**